internal/saucecloud/cloud.go

```diff
@@ -146,7 +148,12 @@ func (r *CloudRunner) collectResults(artifactCfg config.ArtifactDownload, result
 }
 
 func (r *CloudRunner) runJob(opts job.StartOptions) (j job.Job, skipped bool, err error) {
-	log.Info().Str("suite", opts.DisplayName).Str("region", r.Region.String()).Msg("Starting suite.")
+	isRetry := opts.Retries > 0 && opts.Attempt >= 1
+	msg := "Starting suite."
+	if isRetry {
+		msg = "Retrying suite."
+	}
+	log.Info().Str("suite", opts.DisplayName).Str("region", r.Region.String()).Msg(msg)
```

Comment:
Personally I'd move this information to https://github.com/saucelabs/saucectl/pull/404/files#diff-0a4ce723eaf8f4121ec686bc22e0f79d414c6b5bde396ef60b99eab8435bd550R268, since that's where the decision to retry has been made, in which case you can completely remove the `isRetry` check here and https://github.com/saucelabs/saucectl/pull/404/files#diff-0a4ce723eaf8f4121ec686bc22e0f79d414c6b5bde396ef60b99eab8435bd550R186 and just keep everything in one location.

Reply:
:+1: